Builds/CMake/RippledCompiler.cmake

```diff
@@ -28,6 +28,11 @@ target_compile_definitions (common
 if (MSVC)
   # remove existing exception flag since we set it to -EHa
   string (REGEX REPLACE "[-/]EH[a-z]+" "" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+
+  # /ZI (Edit & Continue Debugging Information) is incompatible with Gy-
+  string (REPLACE "/ZI" "/Zi" CMAKE_CXX_FLAGS_DEBUG  ${CMAKE_CXX_FLAGS_DEBUG})
+  string (REPLACE "/ZI" "/Zi" CMAKE_C_FLAGS_DEBUG  ${CMAKE_C_FLAGS_DEBUG})
+
   # also remove dynamic runtime
   foreach (var_
       CMAKE_C_FLAGS_DEBUG CMAKE_C_FLAGS_RELEASE
```

Comment:
Alternatively I could re-purpose this loop to do the same thing

Reply:
slightly simpler if you move your replace inside this loop (presumably it won't match for the release flags anyhow..), so I agree with your suggestion.